src/app.js

```diff
@@ -1,6 +1,8 @@
 import express from "express";
 import "express-async-errors";
 
+import { NamedRouter, routes as registeredRoutes } from "reversical";
```

Comment:
Added "[reversical](https://www.npmjs.com/package/reversical)" for reverse routing